src/index.ts

```diff
@@ -41,17 +50,40 @@ async function start() {
         registerPublicHandlers(SERVICE_NAME, inventoryPublicHandlers);
 
         process.on("SIGINT", () => {
-            console.log("[ITEM-STORE] Gracefully shutting down...");
+            logModel.date = new Date();
+            logger.log({
+                level: 'info',
+                logInfos: logModel,
```

Comment:
what is the purpose of logModel? Could it be added automatically as well by our logger?
I believe the API I'd like is something like:

```
logger.log({
    message: "my message"
});
```

And if I want to do an error:

```
logger.error({
    message: "my error"
});
```


or an info level:

```
logger.info({
    message: "my info"
})
```